src/test/java/teammates/test/cases/util/TimeHelperTest.java

```diff
@@ -40,33 +41,22 @@ public void testFormatTimeForEvaluation() {
     public void testCombineDateTime() {
         String testDate = "01/02/2013";
         String testTime = "0";
-        Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        cal.clear();
-        cal.set(2013, 1, 1, 0, 0, 0);
-        Date expectedOutput = cal.getTime();
+        LocalDateTime expectedOutput = LocalDateTime.of(2013, 2, 1, 0, 0);
 
         testTime = "0";
         ______TS("boundary case: time = 0");
         assertEquals(expectedOutput, TimeHelper.combineDateTime(testDate, testTime));
 
         ______TS("boundary case: time = 24");
         testTime = "24";
-        cal.clear();
-        cal.set(2013, 1, 1, 23, 59);
-        expectedOutput = cal.getTime();
+        expectedOutput = LocalDateTime.of(2013, 2, 1, 23, 59);
         assertEquals(expectedOutput, TimeHelper.combineDateTime(testDate, testTime));
 
         ______TS("negative time");
-        cal.clear();
-        cal.set(2013, 1, 1, -5, 0);
-        expectedOutput = cal.getTime();
-        assertEquals(expectedOutput, TimeHelper.combineDateTime(testDate, "-5"));
```

Comment:
Accepting negative time or >24 hour is a weird feature of `Calendar`. Anyway, we don't need such feature and the new classes don't support them so I just simply consider it as errors